chore(integrate travis CI): update package.json file

Merdoth opened the pull request:
**What does this PR do?**

Integrate TravisCI and add the Travis markdown in the readme file.

**Description of Task to be completed?**

- Setup a .travis.yml file in the root directory.
- Configure the Travis file and include all the settings.
- Add Travis badge to readme file
- Modify readme.md file
- Add settings.json file

**Any background context you want to provide?**

Travis provides continuous integration testing for every code pushed to the repository

**What are the relevant pivotal tracker stories?**

- story type: chore
- story Id: 153009249
- story title: Integrate Travis with readme badge

**Screenshots (if appropriate)**

None

damakahindi commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
```
Why are you adding this line?

temilaj replied:
Totally agree with @damakahindi ! Why?

damakahindi commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cardsforhumanity",
-  "description": "A Game For Horrible People Desperately Trying To Do Good",
+  "description": "A Game For Horrible People Desperately Trying  Do Good",
```
Why are we changing this line?

damakahindi commented on README.md:
```diff
@@ -1,5 +1,8 @@
 [![](https://img.shields.io/badge/Protected_by-Hound-a873d1.svg)](https://houndci.com) [![Coverage Status](https://coveralls.io/repos/github/andela/k2-cfh/badge.svg?branch=chore%2F153009253%2Fintegrate-coveralls-code-coverage-service)](https://coveralls.io/github/andela/k2-cfh?branch=chore%2F153009253%2Fintegrate-coveralls-code-coverage-service)
 
+[![Build Status](https://travis-ci.org/andela/k2-cfh.svg?branch=chore%2F153009249%2Fintegrate-travis-ci)](https://travis-ci.org/andela/k2-cfh)
```
The badge should be on the same line as the other badges.

Commit:
chore(integrate travis CI): update package.json file

- modify readme.md file
- set travis.yml file
- add settings.json file

[Delievers #153009249]
